### PR TITLE
📊  Anthromes dataset

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

File: etl/steps/data/garden/papers/2023-10-20/anthromes.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("anthromes")
+    ds_meadow_input = paths.load_dataset("anthromes_input")
+    # Read table from meadow dataset.
+    tb = ds_meadow["anthromes"].reset_index()
+    tb_input = ds_meadow_input["anthromes_input"].reset_index().drop(columns=["pot_veg", "pot_vll"])
+    # Bit of a hack as the metadata for tb_input is not passed though when opening the shapefile
+    tb_input.metadata = tb.metadata
```
Comment (lucasrodes) [inline etl/steps/data/garden/papers/2023-10-20/anthromes.py:26]: could use 

```suggestion
    tb_input = tb_input.copy_metadata(tb)
```

**Review by lucasrodes — APPROVED** [review]
lgtm! just minor comments

File: snapshots/papers/2023-10-20/anthromes.csv.dvc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    # Citation
+    producer: Gauthier et al.
+    citation_full: |-
+      Gauthier, Nicolas; Ellis, Erle; Klein Goldewijk, Kees, 2021, "Anthromes 12K DGG (V1) Full Dataset", https://doi.org/10.7910/DVN/E3H3AK, Harvard Dataverse
+
+    # Files
+    url_main: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/E3H3AK
+    url_download: https://dataverse.harvard.edu/api/access/datafile/4570057
+    date_accessed: 2023-10-20
```
[the remark this answers is not in the document]
Comment (spoonerf) [reply snapshots/papers/2023-10-20/anthromes.csv.dvc:27]: Ah I actually did start this ages ago and then dropped it, so unfortunately this is actually accurate

[the remark this answers is not in the document]
Comment (spoonerf) [reply snapshots/papers/2023-10-20/anthromes.csv.dvc:27]: :3

**Pull request merged**